### DISCUSSION TITLE
Progress and keep up the good work!!!

### DISCUSSION THREAD

**Discussion by PierreLevres** [Hyundai-Kia-Connect/kia_uvo#237] [Ideas]
I looked at the repo in may last year and decided to stick to my HA integration using bluelinky and code-red. You made great progress since, even borrowing the new stamps mechanism for bluelinky. You might want to give that project credit too :)

I may have overlooked, but is the pypi integration https://github.com/fuatakgun/hyundai_kia_connect_api meant to be inserted into this HA Integration? If so do you want development there or here?

I made a port of bluelinky for Domoticz, so have some missing functions available

**Comment by cdnninja** [comment]
Thanks for the support! That api package exists for a home assistant core pull request that's currently open. 

Right now both live in parallel but if home assistant code accepts it will all shift over.  Right now we are getting some architecture of that repo addressed to support multi car and pain we have had with the current.   Once that's done we may start using it here too while we wait for the core integration.  

I like the idea of adding something referencing bluelinky to the docs.  Feel free to cut a PR or I can get to it when I get a chance.

**Comment by cdnninja** [comment]
I have updated the docs to mention bluelinky.

**Comment by fuatakgun** [comment]
You are right about stamps @PierreLevres , I missed that. Followed your advice here: https://github.com/neoPix/bluelinky-stamps/issues/3
